hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/VirtualdatasourceConfig.java

```diff
@@ -16,10 +16,7 @@
 
 package com.hedera.node.config.data;
 
-import com.hedera.node.config.NetworkProperty;
 import com.swirlds.config.api.ConfigData;
-import com.swirlds.config.api.ConfigProperty;
 
 @ConfigData("virtualdatasource")
-public record VirtualdatasourceConfig(
-        @ConfigProperty(defaultValue = "true") @NetworkProperty boolean jasperdbToMerkledb) {}
+public record VirtualdatasourceConfig() {}
```

Comment:
This class now has no purpose.  Might be better to just remove it entirely.

Reply:
sure, will do.